lasagne/layers/special.py

```diff
@@ -354,3 +355,95 @@ def _meshgrid(height, width):
     ones = T.ones_like(x_t_flat)
     grid = T.concatenate([x_t_flat, y_t_flat, ones], axis=0)
     return grid
+
+
+class ParametricRectifierLayer(Layer):
+    """
+    lasagne.layers.ParametricRectifierLayer(incoming,
+    alpha=init.Constant(0.25), axes=1, **kwargs)
+
+    A layer that applies parametric rectify nonlinearity to its input
+    following [1]_ (http://arxiv.org/abs/1502.01852)
+
+    Equation for the parametric rectifier linear unit:
+    :math:`\\varphi(y_i) = \\max(x_i,0) + \\alpha_i \\min(x_i,0)`
+
+    Parameters
+    ----------
+    incoming : a :class:`Layer` instance or a tuple
+        The layer feeding into this layer, or the expected input shape
+
+    alpha : Theano shared variable, numpy array or callable
+        An initializer for the weights of the layer. If a shared variable or a
+        numpy array is provided the shape should  match the `incoming` shape
+        in all dimension provided by the `axes` parameter and 1 in all others.
+        See :meth:`Layer.create_param` for more information.
+
+    axes : 'auto', 'all', int or tuple of int
+        The axes along which the parameters of the rectifier units are
+        going to be shared. If ``'auto'`` (the default),
+        share over all axes except for the second: this will share biases over
```

Comment:
If you copy/paste, you should still adapt the text :) The ParametricRectifierLayer doesn't have any biases. Also the keyword was `shared_axes` in `BiasLayer`, it should be the same here if the meaning is the same.